tests/unit/test_slurm_ops.py

```diff
@@ -625,6 +646,45 @@ def test_config(self, *_) -> None:
         self.assertEqual(f_info.st_gid, FAKE_GROUP_GID)
 
 
+class TestAcctGatherConfig(TestCase):
+    """Test the `slurmctld` service acct_gather configuration manager."""
+
+    def setUp(self) -> None:
+        self.setUpPyfakefs()
+        self.manager = SlurmctldManager(snap=True)
+        self.fs.create_file(
+            "/var/snap/slurm/common/etc/slurm/acct_gather.conf",
+            contents=EXAMPLE_ACCT_GATHER_CONFIG,
+        )
+
+    def test_config(self) -> None:
```

Comment:
Might be cleaner to split this into multiple tests. There's a lot of asserts in this function.

Reply:
I have it like this as slurmutils should be responsible for testing the nitty gritty of each editor/model with individual test functions. These unit tests are more or less for testing that we don't break the API for managing Slurm configs with slurm_ops.

If anything, we could probably make the tests shorter. The only thing is that we still have to be clever with mocking the uid and gid of the file as pyfakes doesn't provide any abstractions for doing that.